dbt_subprojects/tokens/models/tokens/blast/tokens_blast_erc20.sql

```diff
@@ -13,13 +13,17 @@ SELECT
     , decimals
 FROM (VALUES
     (0x4300000000000000000000000000000000000004, 'WETH', 18)
-    , (0x4300000000000000000000000000000000000003, 'USDB', 18)
```

Comment:
stablecoins table doesn't feed into `tokens.erc20`, so we can't use that as reasoning. however, our automated source does pick this token up, so we can remove safely here:
```
select *
from tokens.erc20
where blockchain = 'blast'
and contract_address = 0x4300000000000000000000000000000000000003
limit 100
```